src/App.js

```diff
@@ -52,14 +53,24 @@ function App() {
             <p className="text-lg text-right">... and still have ${moneyLeft().toLocaleString()} left over.</p>
             }
             
-            <p className="text-base">Elizabeth’s wealth tax, which only impacts America’s 75,000 wealthiest families, would generate enough revenue to cover universal child care, quality public education, forgive student loan debt, provide free public college, and help finance Medicare for All.</p>
+            <p className="text-base">Elizabeth’s <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Ultra-Millionaire Tax</a>, which only impacts America’s 75,000 wealthiest families, funds her biggest and boldest ideas, including:</p>
+              <ul>
+                <li><a href="https://elizabethwarren.com/plans/student-loan-debt-day-one">Student loan debt forgiveness</a></li>
+                <li><a href="https://elizabethwarren.com/kids/">Universal child care</a></li>
+                <li><a href="https://elizabethwarren.com/plans/public-education">Quality public education</a></li>
+                <li><a href="https://elizabethwarren.com/plans/affordable-higher-education">Free public college for all</a></li>
+              </ul>
             <p className="text-base text-center">
-              <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Find out more about the wealth tax</a> and <a href="https://elizabethwarren.com/join-us">join the fight</a>
+              Find out more about the <a href="https://elizabethwarren.com/plans/ultra-millionaire-tax">Ultra-Millionaire Tax</a> and <a href="https://elizabethwarren.com/join-us">join the fight</a>!
             </p>
+
+
+ <p className="text-base text-center">
+              It’s hard to imagine just how much one billion dollars really is—let alone three <i>trillion</i>. To help show how much Warren’s Ultra Millionaire tax could help all Americans, we wanted to show the tangible, human impact of the funds generated from one billionaire alone. Numbers can feel abstract, but behind every number in Warren’s plan are millions of people—andtheir stories.</p>
```

Comment:
This is amazing!

But I see a typo: 

"people—andtheir stories"

Reply:
Yeah, This is a really great addition!

Reply:
thank you!

Reply:
(Will fix the typo in the next round of edits)